.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: csharp
 solution: skalarprodukt.sln
 before_install:
-  - mkdir -p .nuget
-  - wget -O .nuget/nuget.exe https://dist.nuget.org/win-x86-commandline/v4.0.0/nuget.exe
-  - mono .nuget/nuget.exe # Output NuGet's version number
-install:
-  - mono .nuget/nuget.exe restore skalarprodukt.sln -Verbosity detailed
\ No newline at end of file
+  - mkdir ./.paket
+  - wget -O ./.paket/paket.bootstrapper.exe https://github.com/fsprojects/Paket/releases/download/3.18.2/paket.bootstrapper.exe
+  - mono ./.paket/paket.bootstrapper.exe 3.18.2
+  - mono ./.paket/paket.exe restore
+  - mono ./.paket/paket.exe add nuget FSharp.Compiler.Tools version 4.1.0
+  - mono ./.paket/paket.exe install
```

Comment:
I don't think we need to perform that command on CI at all. The purpose of `paket install` is to download the packages and update `csproj` / `fsproj` files to incorporate the new references (if any). That's the developer responsibility to maintain order in the `*proj` files, so `paket install` is a command to be performed on developer machine.

Single `paket restore` should be enough.